content/_index.md

```diff
@@ -3,43 +3,53 @@ title = "Magic Bridges"
 insert_anchor_links = "right"
 +++
 
-# ForgeFlux
+## A day of the past
 
-[ForgeFlux](https://forgeflux.org/) aims to implement federation in [software forges](https://en.wikipedia.org/wiki/Forge_(software)) , enabling
-a decentralized development environment.
+It's a Friday evening, and you've sat down to work on some code. \
```

Comment:
Can you remove `\` from all these documents?

Reply:
That might result in extremely large paragraphs, we cool with that?

The backslashes function as reasonable line breakers.